netman/adapters/switches/brocade.py

```diff
@@ -515,8 +527,8 @@ def __init__(self, *args, **kwargs):
         super(VlanBrocade, self).__init__(*args, **kwargs)
 
         self.vlan_interface_name = kwargs.pop('vlan_interface_name', None)
-        self.tagged_interfaces = []
         self.icmp_redirects = True
+        self.has_no_untagged_interface = False
```

Comment:
instead of saying what the vlan has, maybe you should be saying what it does... 
"has no untagged interface" is miss leading maybe it should be "has a 'no untagged interfaceS' attribute"
and still i'm not sure it's clear
how about vlan.is_default_vlan  or  vlan.is_default_on_interfaces
that would say "here's the state, now with this that i need to..."  instead of "here's an attribute that defines a state that is "?" and will force us to..."


i might be confusing here a bit sorry